articles/portal/ptl-ref-portal-api.md

```diff
@@ -72,8 +72,8 @@ None
 
 #### Example request (Curl)
 
-```
-curl -c /tmp/cookies.txt -X POST -H 'Accept: application/json\' -H 'Content-Type: application/json' -d \'{"email": "email\@example.com", "password": "password"}' https://portal.skyscapecloud.com/api/authenticate
+```bash
+curl -c /tmp/cookies.txt -X POST -H 'Accept: application/json' -H 'Content-Type: application/json' -d '{"email": "email\@example.com", "password": "password"}' https://portal.skyscapecloud.com/api/authenticate
```

Comment:
Probably doesn't actually impact whether or not the code sample works, but I think there's a extraneous escape character before the @ in the email address too.

Reply:
We do need to keep the escape on the '@', else curl interprets it as a filename rather than a string

Reply:
My mistake, a different option was making this happen. Removed the escape character in 710c990